plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/CitadelUtility.java

```diff
@@ -212,6 +215,26 @@ public static boolean attemptReinforcementCreation(Block block, ReinforcementTyp
             Citadel.getInstance().getLogger().info(player.getName() + " created reinforcement with " + type.getName()
                 + " for " + block.getType().toString() + " at " + block.getLocation().toString());
         }
+        // warn if acid cannot complete
+        if (Citadel.getInstance().getAcidManager().isPossibleAcidBlock(block)) {
+            AcidManager acidMan = Citadel.getInstance().getAcidManager();
+            AcidType acidType = acidMan.getAcidTypeFromMaterial(block.getType());
+            for (BlockFace blockFace : acidType.blockFaces()) {
+                Block relativeBlock = block.getRelative(blockFace);
+                Reinforcement relativeReinforcement = ReinforcementLogic.getReinforcementProtecting(relativeBlock);
+                if (
+                    relativeReinforcement == null
+                        || !relativeReinforcement.getType().canBeReinforced(relativeBlock.getType())
```

Comment:
nit: no need for the double indents here, the `) {` being on its own line means a single indent is fine

Reply:
this is just how intellij formats it soz 😭

Reply:
as expected of java tooling